zomes/chat/src/lib.rs

```diff
@@ -71,6 +71,29 @@ fn create_channel(channel_input: ChannelInput) -> ChatResult<ChannelData> {
     channel::handlers::create_channel(channel_input)
 }
 
+#[hdk_extern]
+fn validate(data: ValidateData) -> ExternResult<ValidateCallbackResult> {
```

Comment:
You could also just do a validate_message which only gets called for message entires.

Reply:
tried and it didn't work.  Merging as is.